source/map-filter-reduce.md

```diff
@@ -0,0 +1,47 @@
+# Map, Filter, and Reduce
+
+### Map
+
+The Swift standard library comes with a `map` implementation for the `Array` type. Using `map`, you can obtain
+an new array by applying a function to every member of an existing one.
```

Comment:
I would say it more like "Using `map` you can apply a function to every member of an array to get a new array of the function's return value" or something like that ... basically switch around the "applying function" and "new array" phrasing.